process/test/test_process.py

```diff
@@ -166,6 +166,7 @@ class ProcessCheckTest(AgentCheckTest):
         'system.processes.run_time.avg',
         'system.processes.run_time.max',
         'system.processes.run_time.min',
+        'system.processes.cpu.normalized_pct'
```

Comment:
Can we move this under `system.processes.cpu.pct` just to group them together :)